src/cfclient/ui/dialogs/bootloader.py

```diff
@@ -133,11 +135,11 @@ def __init__(self, helper, *args):
 
         self._platform_filter_checkboxes = []
 
-        self._set_image(self.image_1, "src/cfclient/ui/icons/bolt.webp")
-        self._set_image(self.image_2, "src/cfclient/ui/icons/cf21.webp")
-        self._set_image(self.image_3, "src/cfclient/ui/icons/bl.webp")
-        self._set_image(self.image_4, "src/cfclient/ui/icons/flapper.webp")
-        self._set_image(self.image_5, "src/cfclient/ui/icons/tag.webp")
+        self._set_image(self.image_1, ICON_PATH + "/bolt.webp")
```

Comment:
Use `os.path.join` for cross platform paths, also avoids problems with slashes.

```suggestion
        self._set_image(self.image_1, os.path.join(ICON_PATH, "bolt.webp"))
```